src/main/java/homework_2/random_chars_table/TablePrinter.java

```diff
@@ -0,0 +1,46 @@
+package homework_2.random_chars_table;
+
+import java.util.LinkedList;
+import java.util.List;
+
+class TablePrinter {
+
+    private final int rows;
+    private final int columns;
+    private final int remains;
+    private final String strategy;
+    private List<Character> charList;
+
+    public TablePrinter(int rows, int columns, String strategy) {
+        this.rows = rows;
+        this.columns = columns;
+        this.strategy = strategy;
+        remains = "even".equalsIgnoreCase(strategy) ? 0 : 1;
+    }
+
+    public void printTable() {
+        charList = new LinkedList<>();
+        for (int i = 0; i < rows; i++) {
+            System.out.print("| ");
```

Comment:
Please remove whitespace, it fail my tests. In AC the output is without whitespaces

Reply:
Done